src/pathutils.rs

```diff
@@ -0,0 +1,25 @@
+use crate::shell::Shell;
+use std::process::Command;
+
+#[cfg(windows)]
+pub fn format_path(path: &str, shell: &str) -> String {
+    match shell {
+        "cmd" | "pwsh" => String::from(path),
+        _ => {
+            let output = Command::new("cygpath")
+                .arg(path)
+                .output()
+                .expect("failed to convert Windows path to UNIX");
```

Comment:
that's cool!!!!

1. can we make the error message a little clearer so error reports will be easier for me? `Failed to convert Windows path to Unix: (actual error message)` -- most users would probably don't know that it's `cygpath` which generates the paths
2. can we make this function `std::io::Result<String>` and handle the `Err` branch in the callsite?

Reply:
I can add anyhow context to display the actual error message.

For the second point, how would the error even be handled? I don't really see a way to proceed if the path can't be converted properly.

Reply:
We can’t continue but we don’t want to panic: we can rely on error handling